common.py

```diff
@@ -50,13 +50,21 @@ def write(self, msg: str) -> None:
             return
         if not self._logfile:
             print(f"{ADDON_NAME_SHORT} debug: opening log file {DEBUG_LOG_FILE_PATH}")
+            # clear before writing to avoid large logfiles and their consequences
+            open(DEBUG_LOG_FILE_PATH, 'w').close()
             self._logfile = open(DEBUG_LOG_FILE_PATH, "a")
```

Comment:
> In that case we should maybe build in something like splitted logfiles, a button to clear the log file or making the view opened in "Open Logfile" editable. It is a bit of a pain if you are using anki for a long time and it needs a few moments to cool down and close the log.

The existing logging code was written by Russel Simmons a long time ago. Later I refactored it a bit, but the logic is mostly still the same. Today we should just do:

```
from aqt.addons import AddonManager
logger = AddonManager.get_logger(__name__)
logger.info("my message")
```

If we use Anki's logging, it will automatically clear old logs (older than 10 days by default).